db/migrate/20200304144205_create_prosecution_case_defendant_offences.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateProsecutionCaseDefendantOffences < ActiveRecord::Migration[6.0]
+  def change
+    create_table :prosecution_case_defendant_offences, id: :uuid do |t|
+      t.string :prosecution_case_id,    null: false
+      t.string :defendant_id,           null: false
```

Comment:
Should we have these as `uuid`s instead?
also, perhaps one for later when we loop: index for `defendant_id`

Reply:
Changed to UUIDs